tests/ctst/features/quotas/Quotas.feature

```diff
@@ -96,3 +96,43 @@ Feature: Quota Management for APIs
             | 100        | 200         | 0            | IAM_USER |
             | 100        | 0           | 200          | IAM_USER |
             | 100        | 200         | 200          | IAM_USER |
+
+    @2.6.0
+    @PreMerge
+    @Quotas
+    @CronJob
+    @DataDeletion
+    @NonVersioned
+    Scenario Outline: Quotas are affected by deletion operations between count items runs
+        Given an action "DeleteObject"
+        And a permission to perform the "PutObject" action
+        And a STORAGE_MANAGER type
+        And a bucket quota set to 10000 B
+        And an account quota set to 10000 B
+        And an upload size of 1000 B for the object "obj-1"
+        And a bucket quota set to <bucketQuota> B
+        And an account quota set to <accountQuota> B
+        And a <userType> type
+        And an environment setup for the API
+        And an "existing" IAM Policy that "applies" with "ALLOW" effect for the current API
+        When I wait 3 seconds
+        And I PUT an object with size <uploadSize>
+        Then the API should "fail" with "QuotaExceeded"
+        When the "count-items" cronjobs completes without error
+        When I wait 3 seconds
+        # At this point if negative inflights are not supported, write should
+        # not be possible, as the previous inflights are now part of the current
+        # metrics.
+        And i delete object "obj-1"
+        And I wait 3 seconds
+        And I PUT an object with size <uploadSize>
+        Then the API should "succeed" with ""
+
+        Examples:
+            | uploadSize | bucketQuota | accountQuota | userType |
+            | 100        | 200         | 0            | ACCOUNT  |
+            | 100        | 0           | 200          | ACCOUNT  |
+            | 100        | 200         | 200          | ACCOUNT  |
+            | 100        | 200         | 0            | IAM_USER |
+            | 100        | 0           | 200          | IAM_USER |
+            | 100        | 200         | 200          | IAM_USER |
```

Comment:
not sure where to put this feedback on [`development/2.8...w/2.8/improvement/ZENKO-4941`#diff-0f9a8cc11d](https://github.com/scality/Zenko/compare/development/2.8...w/2.8/improvement/ZENKO-4941#diff-0f9a8cc11d2560e9e9ea8668d53be732997ae7cecb4ec7201c836bc8367fc5c9R175)

→ should be tagged as `@2.6.0`, since it should work with zenko 2.6.0 as well (the restriction is in our implementation of tests only)

Reply:
Keeping this open, will do it when we have the right projects versions (scuba, mostly) to save time